src/app/components/ConsensusTransactionMethod/index.tsx

```diff
@@ -75,19 +99,93 @@ export const MethodIcon: FC<MethodIconProps> = ({
       >
         {cloneElement(icon, { sx: { fontSize: Math.ceil(size * iconRatio) } })}
       </Box>
-      {label && <Typography sx={{ textTransform: 'capitalize' }}>{label}</Typography>}
+      {label && (
+        <Typography
+          sx={{
+            fontWeight: 'inherit',
+            textTransform: 'capitalize',
+            order: reverseLabel ? -1 : 0,
+            ...(truncate
+              ? {
+                  minWidth: '100px',
+                  maxWidth: '1px',
```

Comment:
This is unexpected. Typo? Add a comment?

Reply:
will add a comment

Reply:
`// all properties are needed to enable overflow, maxWidth allows to squeeze content`
doesn't really explain it for me. Simply `maxWidth: '100px'` works the same?